guide/plg_atTheTelescope.tex

```diff
@@ -837,11 +850,43 @@ \section{INDI}
 \item IOptron GotoNova Upgrade Kit 8400
 \end{itemize}
 
+\section{ASCOM}
+\label{sec:plugins:TelescopeControl:ASCOM}
+
+\indexterm{ASCOM}, \newFeature{v0.19.3}is probably the most widely used way how to communicate with telescopes, mounts
+and other astronomy equipment. The ASCOM platform aims to decouple device dependent specifics by introducing a common
+standard for communication between device drivers and applications. Almost every telescope mount comes with an ASCOM 
+driver and most of the astronomy software and tools can communicate with ASCOM compatible devices.
+
+The ASCOM telescope support has been added to Stellarium's Telescope Control plugin by Gion Kunz in version 0.19.3.
+
+Note that in order for Stellarium to communicate with your ASCOM compatible mount, you will need to have the ASCOM
+plaform installed on your Windows machine (version 6 or higher). Also, the ASCOM driver of your telescope mount
+needs to be installed before you can start using the telescope in Stellarium.
+
+\textbf{Important:} Most modern ASCOM drivers are written to run in 32bit and 64bit mode (any) and therefore are 
+compatible with the 64bit and 32bit version of Stellarium. However, if you happen to have a 32bit or 64bit only
+driver for your telescope mount, you will also need to install the respective Stellarium version.
+
+In order to use your ASCOM telescope you need to select the correct device in the ASCOM chooser by clicking
+the button \button{Choose ASCOM telescope}. The selected device is shown below that button.
+
+By default, Stellarium is trying to ask the ASCOM driver in order to determine if JNow or J2000 coordinates 
+should used for communicating with the mount. If you'd like to override this behavior and tell the ASCOM telescope
+to communicate in the coordinate system configured in the section \nameref{sec:plugins:TelescopeProperties}, you can
+switch the radio button \button{Let ASCOM device decide} to \button{Use Stellarium settings} in the ASCOM settings of 
+the Configure Telescope dialog.
+
 %\newpage
 \section{StellariumScope}
 \label{sec:plugins:StellariumScope}
 \program{StellariumScope} is a free add-on that enables you to control your telescope with Stellarium. 
 
+Note that Stellarium supports ASCOM telescopes natively since version 0.19.3. There's no need to 
+  use Stellarium Scope in order to use your ASCOM telescopes with Stellarium anymore. We are 
```

Comment:
I think correct spelling is StellariumScope? (2x)

Reply:
Fixed